public/src/card/sectioned-card-list.html

```diff
@@ -0,0 +1,211 @@
+<link rel="import" href="../../bower_components/polymer/lib/elements/dom-repeat.html">
+<link rel="import" href="card-list.html">
+
+<dom-module id="sectioned-card-list">
+  <template>
+    <style is="custom-style" include="app-styles">
+      :host {
+        display: block;
+        box-sizing: border-box;
+      }
+
+      card-list {
+        max-width: 1264px;
+        margin: 10px auto;
+        box-sizing: border-box;
+      }
+
+      .promotedPanel {
+        background: #fafafa;
+        padding: 2px 0 10px;
+        border-top: 1px solid #f0f0f0;
+        border-bottom: 1px solid #f0f0f0;
+      }
+
+      #more {
+        box-sizing: border-box;
+        padding: 30px 0 20px;
+      }
+
+      #moreButton {
+        text-align: center;
+        width: 85px;
+        height: 85px;
+        cursor: pointer;
+        margin: 0 auto;
+        border-radius: 50%;
+        font-size: 16px;
+        text-transform: uppercase;
+        font-weight: bold;
+        letter-spacing: 0.05em;
+        color: var(--highlight-green);
+        background: transparent;
+        border: 3px solid;
+      }
+
+      @media (max-width: 1264px) {
+        card-list {
+          max-width: 948px;
+        }
+      }
+
+      @media (max-width: 948px) {
+        card-list {
+          max-width: 632px;
+        }
+      }
+
+      @media (max-width: 632px) {
+        card-list {
+          max-width: 320px;
+        }
+      }
+    </style>
+    <dom-repeat items="[[sections]]">
+      <template>
+        <div class$="{{getPanelClass(item)}}">
```

Comment:
Any reason not to just put the class as one of the section members?

Reply:
No reason. I tend to keep style related items out of the data. Just a habit. Could have been done either way.